Packs/Anomali_ThreatStream/Playbooks/playbook-Detonate_URL_-_ThreatStream.yml

```diff
@@ -293,212 +331,119 @@ inputs:
       root: URL
```

Comment:
maybe we should point directly to the `URL.Data` instead of just 'URL'

Reply:
done. also added the 'unique' transformer to the input